lua/starfall/libs_sh/players.lua

```diff
@@ -397,6 +397,43 @@ if CLIENT then
 	function player_methods:voiceVolume()
 		getply(self):VoiceVolume()
 	end
+	
+	--- Animates a hologram
+	-- @client
+	-- @param animation sequence string or act number. https://wiki.facepunch.com/gmod/Enums/ACT
+	-- @param loop Should the gesture loop
+	-- @param slot The gesture slot to use. GESTURE_SLOT table values
+	-- @param weight The weight of the gesture
+	function player_methods:playGesture(animation, loop, slot, weight)
+		slot = slot or GESTURE_SLOT_CUSTOM
+		checkluatype(slot, TYPE_NUMBER)
```

Comment:
if slot==nil then slot = GESTURE_SLOT_CUSTOM else checkluatype(slot, TYPE_NUMBER) end

Reply:
The documentation should also specify that these parameters are optional.